src/main/java/morty/Morty.java

```diff
@@ -0,0 +1,43 @@
+package morty;
+
+import morty.command.Command;
+
+public class Morty {
```

Comment:
Maybe include header comments for classes and public methods to better follow the coding standard guidelines? I noticed this in several other places.